common/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id "org.sonarqube" version "2.5"
+    id "org.sonarqube" version "2.6.2"
```

Comment:
Is this dependency needed for `jacocoTestReport` ?

Reply:
No its not. I have updated to the latest version available since we update gradle to latest